vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -56,37 +68,57 @@ function Get-FilesToCacheOnVHD
 
     $map = @{
         "c:\akse-cache\" = @(
+            # TODO(mainred): change this path to master branch AgentBaker after this is merged into master
```

Comment:
Remove all TODO in this file?

Reply:
Will remove the TODO, but we may ignore the windows vhd check because of master branch has no such file until this is merged.